.github/workflows/nodejs.yml

```diff
@@ -11,4 +11,4 @@ jobs:
     # Documentation: https://github.com/zakodium/workflows#nodejs-ci
     uses: zakodium/workflows/.github/workflows/nodejs.yml@nodejs-v1
     with:
-      node-version-matrix: '[12, 14, 16]'
+      node-version-matrix: '[14, 16]'
```

Comment:
node 12 is still supported, we should keep it if possible

Reply:
fixed